src/sql/parts/dashboard/dashboardTab.contribution.ts

```diff
@@ -4,27 +4,94 @@
  *--------------------------------------------------------------------------------------------*/
 import { IExtensionPointUser, ExtensionsRegistry } from 'vs/platform/extensions/common/extensionsRegistry';
 import { IJSONSchema } from 'vs/base/common/jsonSchema';
+import { localize } from 'vs/nls';
 
 import { GenerateDashboardWidgetSchema } from 'sql/parts/dashboard/pages/dashboardPageContribution';
-import { IDashboardTabContrib, RegisterTab } from 'sql/platform/dashboard/common/dashboardRegistry';
+import { RegisterTab } from 'sql/platform/dashboard/common/dashboardRegistry';
+import { WidgetConfig } from 'sql/parts/dashboard/common/dashboardWidget';
+
+export interface IDashboardTabContrib {
```

Comment:
Why do you have both IDashboardTabContrib  and IDashboardTab?

Reply:
One is the interface for the object we get from the contribution and the other is the one that we actually store,.